src/utils/NotificationUtils.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+import * as admin from 'firebase-admin';
+// eslint-disable-next-line import/no-unresolved
+import { getMessaging } from 'firebase-admin/messaging';
+
+const schedule = require('node-schedule');
+
+function sendNotification(deviceToken: string, notifData) {
+  const message = {
+    data: notifData,
+    token: deviceToken,
+  };
+  if (deviceToken !== '') {
+    getMessaging().send(message)
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
+}
+
+function waitForDeparture(deviceToken: string, startTime: string) {
+  const startDate = new Date((parseInt(startTime) * 1000) - (60000 * 10));
+
+  const notifData = {
+    data: 'You should board your bus in 10 minutes',
+    notification: 'testBody',
+  };
+  schedule.scheduleJob(startDate,
```

Comment:
rn I think this schedules the notification to be sent only on the startDate, should we change it to daily? i think node-schedule allows you to us cron logic to do that

Reply:
I think we aren't going to implement recurring notifications just yet, unless you think this would be a good idea to do right now?

Reply:
nah i think this is a good POC and then we can extend later